presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -262,6 +262,14 @@ Property Name                                      Description
 
 ``hive.file-status-cache-expire-time``             How long a cached directory listing should be considered     ``1m``
                                                    valid.
+
+``hive.sorted-writing-temp-path``                  Directory used for temporary files while writing bucketed
```

Comment:
Why not to use `hive.temporary-staging-directory-path`?

Reply:
We could use it, though we would need to introduce a boolean toggle `hive.sorted-writing-temporary-staging-enabled`  or would we tie it to existing flag `hive.temporary-staging-directory-enabled` ? 
`hive.temporary-staging-directory-enabled` is enabled by default and `hive.temporary-staging-directory-path` is also set to `/tmp/presto-${USER}` by default.

Reply:
>  or would we tie it to existing flag hive.temporary-staging-directory-enabled ?

That would my preference. Why would you like to have it separate?

> hive.temporary-staging-directory-enabled is enabled by default and hive.temporary-staging-directory-path is also set to /tmp/presto-${USER} by default.

It sounds like a good default value for bucketed tables as well.

Reply:
I think enabling it would at least double the storage requirements from the current requirement for staging S3 writes. So having a separate on/off toggle would be good. I'll make the change to use the same directory location though, I don't see any issue with that.
I'm inclined to enable the toggle by default but it risks failures in some cases https://prestosql.slack.com/archives/CGB0QHWSW/p1576613895090800?thread_ts=1576180076.157800&cid=CGB0QHWSW

Reply:
Made changes to add `hive.sorted-writing-temp-staging-path-enabled`, it's disabled by default and the path used is `hive.temporary-staging-directory-path` when the flag is enabled.

Reply:
> I think enabling it would at least double the storage requirements from the current requirement for staging S3 writes.

That highly depends on use case. I am under the impression that not many people are doing writes to sorted bucketed tables, otherwise we would get more complains. If I am correct, then most people won't notice a change. On the other hand you you are using sorted tables much frequently than other tables then it also does not matter.

My point is that having two configuration toggles for very similar things makes it more complicated to maintain the Presto installation.  Maybe we should use single toggle for both cases and fallback if we hit some problem? WDYT?

@electrum Do you have any thoughts for that?

Reply:
I'm ok to enable `hive.sorted-writing-temp-staging-path-enabled` by default but I think there may still be a case to retain a separate flag as the 2 flags are not exactly the same thing. Disabling `hive.sorted-writing-temp-staging-path-enabled` results in the existing behavior of using target path for staging sorted writes instead of a local path. But disabling `hive.temporary-staging-directory-enabled` results in direct writes to target path with no staging. 
I'm concerned that if we have a common flag and if someone wants to disable staging of sorted writes on local path, then they would also be forced to use direct writes for non-sorted table cases.
If there is agreement that this is not big a deal, I'm ok to use just one flag as well.

Reply:
I've made changes to reuse `hive.temporary-staging-directory-enabled` and removed `hive.sorted-writing-temp-staging-path-enabled` now